beanie/odm/fields.py

```diff
@@ -25,7 +25,9 @@
 
 from bson import DBRef, ObjectId
 from bson.errors import InvalidId
-from pydantic import BaseModel
+from pydantic import (
+    BaseModel,
```

Comment:
Unnecessary trailing comma

Reply:
IMHO this should be configured and  fixed by ruff on commit

Reply:
removed and this should be caught by ruff

Reply:
> IMHO this should be configured and fixed by ruff on commit

Sounds good, man. Happy to review the PR for it.

Reply:
will do when i get time for it